crates/livesplit-title-abbreviations/src/lib.rs

```diff
@@ -214,3 +220,71 @@ pub fn abbreviate_category(category: &str) -> Vec<Box<str>> {
 
     abbrevs
 }
+
+#[cfg(test)]
+mod tests {
+    use super::abbreviate;
+    use alloc::boxed::Box;
+    use alloc::vec;
+
+    #[test]
+    fn burnout_3() {
+        let abbreviations = abbreviate("Burnout 3: Takedown");
+
+        let expected = vec![
+            Box::from("B3"),
+            Box::from("B3: Takedown"),
+            Box::from("Burnout 3"),
+            Box::from("Takedown"),
+            Box::from("Burnout 3: Takedown"),
+        ];
+
+        assert_eq!(abbreviations, expected);
+    }
+
+    #[test]
+    fn tww() {
+        let abbreviations = abbreviate("The Legend of Zelda: The Wind Waker");
+
+        let expected = vec![
+            Box::from("Legend of Zelda: TWW"),
+            Box::from("Legend of Zelda: The Wind Waker"),
+            Box::from("Legend of Zelda: Wind Waker"),
+            Box::from("TLoZ: TWW"),
+            Box::from("TLoZ: The Wind Waker"),
+            Box::from("TLoZ: Wind Waker"),
+            Box::from("TWW"),
+            Box::from("The Wind Waker"),
+            Box::from("Wind Waker"),
+            Box::from("The Legend of Zelda: The Wind Waker"),
+        ];
+
+        assert_eq!(abbreviations, expected);
+    }
+
+    #[test]
+    fn bfbb() {
+        let abbreviations = abbreviate("SpongeBob SquarePants: Battle for Bikini Bottom");
+
+        let expected = vec![
+            Box::from("Battle for Bikini Bottom"),
+            Box::from("BfBB"),
+            Box::from("SS: Battle for Bikini Bottom"),
+            Box::from("SS: BfBB"),
+            Box::from("SpongeBob SquarePants: Battle for Bikini Bottom"),
+        ];
+
+        assert_eq!(abbreviations, expected);
+    }
+    #[test]
+    fn sm64() {
+        let abbreviations = abbreviate("Super Mario 64");
+
+        let expected = vec![
+            Box::from("SM64"),
+            Box::from("Super Mario 64")
```

Comment:
apparently `rustfmt` wants these to be on a single line, which arguably looks worse imo

Reply:
There's #[rustfmt::skip] you can put, if you really don't want to take its suggestion.